lib/slider/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import Slider from 'react-slick';
-import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
+// import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
```

Comment:
these arrows don't seem to be working... It cause an 'invalid' violation in react so I commented them out. Do you know why they are not working?